### PR TITLE
atc: remove objectId from locks held metric [#2980]

### PR DESCRIPTION
**Pull request by cirocosta**
tl;dr: this PR removes `objectId` from `locksheld` metric, but I *think* this metric *might* be more useful if we change it to keep track of lock acquiring & releasing durations.

---

Hey,

In this PR I try to address the problems that the *very* high cardinality of the [`LocksHeld`](https://github.com/concourse/concourse/pull/2681) might present by completely removing `objectId` from the final metric (which would get rid of that from all emitters - manually done for Prometheus).

```sh
curl -s localhost:1337/metrics | grep locks
# HELP concourse_locks_state Database locks held
# TYPE concourse_locks_state gauge
concourse_locks_held{type="Batch"} 0
concourse_locks_held{type="DatabaseMigration"} 0
```

While I see why one would like to have the `ObjectID` included (we could be able to see when the lock for "CreatingContainer" where `containerID=1` was acquired and when released), I'm not very sure whether having the metric without such dimension (`objectid`) is still all that relevant.

*ps.: If the intention of the metric was to answer the question of "are we having locks not being released for too long?", maybe a different type of metric could be used - one like `Histogram` where we could keep track of the duration from `acquire` to `release` for each lock type (keeping track of the individual ObjectID locks in-memory only).*

Wdyt?

thx!

cc @topherbullock

### CODE REVIEW

**Comment by cirocosta** [comment]
Thanks!

Just added a new panel to hush-house that consumes it in a per-ATC manner:

screen shot 2019-01-17 at 8 12 15 am
